src/main/java/com/mucommander/text/Translator.java

```diff
@@ -222,36 +220,14 @@ public static String get(String key, String... paramValues) {
          */
         ResolveVariableResourceBundle(final ResourceBundle resourceBundle) {
             this.resourceBundle = resourceBundle;
+            this.cache = ResolveVariableResourceBundle.resolve(resourceBundle);
         }
 
         @Override
         protected Object handleGetObject(final String key) {
-            Object result = cache.get(key);
+            final Object result = cache.get(key);
             if (result == null) {
-                result = resourceBundle.getObject(key);
-                if (result instanceof String) {
-                    final String value = (String) result;
-                    final Matcher matcher = VARIABLE.matcher(value);
-                    if (matcher.find()) {
-                        int startIndex = 0;
-                        final StringBuilder buffer = new StringBuilder(64);
-                        while (matcher.find(startIndex)) {
-                            buffer.append(value, startIndex, matcher.start());
-                            try {
-                                buffer.append(handleGetObject(matcher.group(1)));
-                            } catch (MissingResourceException e) {
-                                if (LOGGER.isDebugEnabled()) {
-                                    LOGGER.debug("The key '%s' is missing", key);
-                                }
-                                buffer.append(value, matcher.start(), matcher.end());
-                            }
-                            startIndex = matcher.end();
-                        }
-                        buffer.append(value.substring(startIndex));
-                        result = buffer.toString();
-                        cache.putIfAbsent(key, (String) result);
-                    }
-                }
+                return resourceBundle.getObject(key);
```

Comment:
@ahadas Actually we still need it as it only keep key/value pairs whose value contains at least one variable as you can see [here](https://github.com/mucommander/mucommander/pull/17/files#diff-1517cfcc43a86b331edf0ab843b6b0b1R280)